app/src/modules/content/routes/item.vue

```diff
@@ -57,6 +57,18 @@
 		</template>
 
 		<template #actions>
+			<v-button
+				v-if="previewURL"
+				v-tooltip.bottom="t(livePreviewMode === null ? 'live_preview.enable' : 'live_preview.disable')"
+				rounded
+				icon
+				class="action-preview"
+				:secondary="livePreviewMode === null"
+				@click="toggleSplitView"
+			>
+				<v-icon :name="livePreviewMode === null ? 'visibility' : 'visibility_off'" outline />
+			</v-button>
```

Comment:
I think this button should be hidden for tablet and smaller screen sizes. While you can trigger the split screen on smaller screens currently this does not work. 
![image](https://github.com/directus/directus/assets/9389634/9b77c5e4-d902-45b7-aaa1-c3c7fa7238a2)